ci/pipelines/deploy-to-test.yml

```diff
@@ -553,6 +560,9 @@ groups:
       - deploy-toolbox
       - smoke-test-toolbox
       - push-toolbox-to-staging-ecr
+  - name: webhooks
+    jobs:
+      - deploy-webhooks
```

Comment:
In theory there's nothing to stop you adding a `smoke-test-webhooks` here (and copy-paste the definition further down obv). For example we run the Canaries against test-12 after Toolbox release, even though they don't have coverage for Toolbox. Then if something weird *does* happen, at least you'll get a notification...

Reply:
think we'll leave these off just for now - as you say, easy to add in when needed